_includes/payment-orders-resource.md

```diff
@@ -0,0 +1,277 @@
+            "rel": "view-paymentorder",
+            "method": "GET",
+            "contentType": "application/javascript"
+        }
+    ]
+}
+```
+
+{:.table .table-striped}
+| Field          | Type     | Description                                                                        |
+| :------------- | :------- | :--------------------------------------------------------------------------------- |
+| `paymentorder` | `object` | The payment order object.                                                          |
+| └➔&nbsp;`id`   | `string` | {% include field-description-id.md resource="paymentorder" %}                      |
+| `operations`   | `array`  | The array of possible operations to perform, given the state of the payment order. |
+
+The `paymentorder` object is abbreviated since it's just the `id` and
+`operations` we are interested in. Store the `id` of the Payment Order
+in your system to look up status on the completed payment later.
+
+Then find the `view-paymentorder` operation and embed its `href` in a
+`<script>` element.
+That script will then load the Seamless View for the Payment Menu. We will look
+into how to hook that up next.
+
+{% include alert.html type="informative" icon="info" body=" `orderReference` must
+be sent as a part of the `POST` request to `paymentorders` and must represent
+the order ID of the webshop or merchant website." %}
+
+## Step 2: Display the Payment Menu
+
+To load the payment menu from the JavaScript URL obtained in the back end API
+response, it needs to be set as a `script` element's `src` attribute. You can
+cause a page reload and do this with static HTML or you can avoid the page
+refresh by invoking the POST to create the payment order through Ajax and then
+create the script element with JavaScript, all inside the event handler for
+[`onConsumerIdentified`][technical-reference-onconsumer-identified].
+The HTML code will be unchanged in this example.
+
+{:.code-header}
+**JavaScript**
+
+```js
+var request = new XMLHttpRequest();
+request.addEventListener('load', function () {
+    // We will assume that our own backend returns the
+    // exact same as what SwedbankPay returns.
+    response = JSON.parse(this.responseText);
+    var script = document.createElement('script');
+    // This assumes the operations from the response of the POST of the
+    // payment order is returned verbatim from the server to the Ajax:
+    var operation = response.operations.find(function (o) {
+        return o.rel === 'view-consumer-identification';
+    });
+    script.setAttribute('src', operation.href);
+    script.onload = function () {
+        payex.hostedView.consumer({
+            // The container specifies which id the script will look for
+            // to host the checkin component
+            container: 'checkin',
+            onConsumerIdentified: function onConsumerIdentified(consumerIdentifiedEvent) {
```

Comment:
`onConsumerIdentified` is not in use when not doing Checkin. We need to rewrite this entire code example to work with Payment Menu.